akka-docs/rst/java/scheduler.rst

```diff
@@ -15,6 +14,22 @@ You can schedule sending of messages to actors and execution of tasks
 (functions or Runnable).  You will get a ``Cancellable`` back that you can call
 :meth:`cancel` on to cancel the execution of the scheduled operation.
 
+The scheduler in Akka is designed for high-throughput of thousands up to millions 
+of triggers. The prime use-case being triggering Actor receive timeouts, Future timeouts,
+circuit breakers and other time dependent events which happen all-the-time and in many 
+instances at the same time. The implementation is based on a Hashed Wheel Timer, which is
+a known datastructure and algorithm for handling such use cases, refer to the `Hashed and Hierarchical Timing Wheels`_ 
+whitepaper by Varghese and Lauck if you'd like to understand its inner workings. 
+The implementation in Akka was initially inspired by the one however added a number of optimisations 
+since, some of which were ported back to Netty and other implementations. 
+
+The Akka scheduler is **not** designed for long-term scheduling (see `akka-quartz-scheduler`_ 
```

Comment:
"see for example: akka-quartz-scheduler" (another case of favoring one alternative when there may be many - we are not a framework/opinionated and all that?)